docs/installation/integrations/react.md

```diff
@@ -205,40 +206,40 @@ If you use the {@link framework/document-editor document (decoupled) editor}, yo
 import DecoupledEditor from '@ckeditor/ckeditor5-build-decoupled-document';
 
 class App extends Component {
-    editor = null;
-
-    render() {
-        return (
-            <div className="App">
-                <h2>CKEditor&nbsp;5 using a custom build - decoupled editor</h2>
-                <CKEditor
-                    onReady={ editor => {
-                        console.log( 'Editor is ready to use!', editor );
-
-                        // Insert the toolbar before the editable area.
-                        editor.ui.getEditableElement().parentElement.insertBefore(
-                            editor.ui.view.toolbar.element,
-                            editor.ui.getEditableElement()
-                        );
-
-                        this.editor = editor;
-                    } }
-                    onError={ ( error, { willEditorRestart } ) => {
-                        // If the editor is restarted, the toolbar element will be created once again.
-                        // The `onReady` callback will be called again and the new toolbar will be added.
-                        // This is why you need to remove the older toolbar.
-                        if ( willEditorRestart ) {
-                            this.editor.ui.view.toolbar.element.remove();
-                        }
-                    } }
-                    onChange={ ( event ) => console.log( event ) }
-                    editor={ DecoupledEditor }
-                    data="<p>Hello from CKEditor&nbsp;5's decoupled editor!</p>"
-                    config={ /* the editor configuration */ }
-                />
-            </div>
-        );
-    }
+	editor = null;
+
+	render() {
+		return (
+			<div className="App">
+				<h2>CKEditor&nbsp;5 using a custom build - decoupled editor</h2>
+				<CKEditor
+					onReady={ editor => {
+						console.log( 'Editor is ready to use!', editor );
+
+						// Insert the toolbar before the editable area.
+						editor.ui.getEditableElement().parentElement.insertBefore(
+							editor.ui.view.toolbar.element,
+							editor.ui.getEditableElement()
+						);
+
+						this.editor = editor;
+					} }
+					onError={ ( error, { willEditorRestart } ) => {
+						// If the editor is restarted, the toolbar element will be created once again.
+						// The `onReady` callback will be called again and the new toolbar will be added.
+						// This is why you need to remove the older toolbar.
+						if ( willEditorRestart ) {
+							this.editor.ui.view.toolbar.element.remove();
+						}
+					} }
```

Comment:
```suggestion
					onAfterDestroy={ editor => {
						editor.ui.view.toolbar.element.remove();
					} }
```

Reply:
I'm not sure if we should include this `onAfterDestroy` example with `editor.ui.view.toolbar.element.remove();`. It really depends on the way the users mounted `toolbar`. If it's somewhere outside of React box then it's fine but if it's in div that is managed by React then it will lead to crashes, probably. I'll leave there a comment that only external things should be removed here.

Reply:
For more context: this code sample is for Decoupled Editor. This editor is characteristic in the way that it is the integrator's responsibility to insert the toolbar where they want to.